### PR TITLE
Feat/product lists

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added product listing page
Screenshot (30)

### CODE REVIEW

**Review by devdeadviz — APPROVED** [review]
Remove commented code if it's not required. Other things look good to me!

**Comment by devdeadviz** [comment]
Add more spacing between filter and product section.

File: Components/Products/product.html [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+<body>
+    <header class="navbar-home">
+        <nav class="navbar-wrapper">
+          <div class="navbar-left-aligned">
+            <svg
+              class="sidebar-toggler"
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
```
Comment (haarsh24) [inline Components/Products/product.html:27]: why are you using these properties in html ?

File: Components/zia.css [hunk 2]
```diff
@@ -28,6 +28,7 @@ body {
     --very-light-primary-color: #e8ffdb;
   
     --complimentary-text-color: #f5ece5;
+    --transparent-white: white;
```
Comment (haarsh24) [inline Components/zia.css:31]: just a suggestion to use the same color code for everything.

**Review by haarsh24 — APPROVED** [review]
Great work with UI .
You can merge.

File: Components/zia.css (hunk 2 above)
Comment (SHIKHASINGH1506) [reply Components/zia.css:31]: I'm using same text color everywhere . Few places like on primary buttons and all.

**Pull request merged**